Dockerfile

```diff
@@ -1,38 +1,32 @@
+ARG IRITYPE=cliri
+
 FROM iotacafe/maven:3.5.4.oracle8u181.1.webupd8.1.1-1 as local_stage_build
 MAINTAINER giorgio@iota.org
-
-WORKDIR /iri
-
-COPY . /iri
+ARG IRITYPE
+WORKDIR /${IRITYPE}
+COPY . /${IRITYPE}
 RUN mvn clean package
 
 # execution image
-FROM iotacafe/java:oracle8u181.1.webupd8.1-1
-
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        jq curl socat \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY --from=local_stage_build /iri/target/iri*.jar /iri/target/
-COPY docker/entrypoint.sh /
-
-# Java related options. Defaults set as below
-ENV JAVA_OPTIONS="-XX:+UnlockExperimentalVMOptions -XX:+DisableAttachMechanism -XX:InitiatingHeapOccupancyPercent=60 -XX:G1MaxNewSizePercent=75 -XX:MaxGCPauseMillis=10000 -XX:+UseG1GC"
-ENV JAVA_MIN_MEMORY 2G
-ENV JAVA_MAX_MEMORY 4G
-
-# Additional custom variables. See DOCKER.md for details
-ENV DOCKER_IRI_JAR_PATH "/iri/target/iri*.jar"
-ENV DOCKER_IRI_REMOTE_LIMIT_API "interruptAttachToTangle, attachToTangle, addNeighbors, removeNeighbors, getNeighbors"
-
+#FROM iotacafe/java:oracle8u181.1.webupd8.1-1
+FROM openjdk:jre-slim
+ARG IRITYPE
+ENV DOCKER_IRI_JAR_PATH=/${IRITYPE}/target/${IRITYPE}*.jar \
+    DOCKER_IRI_REMOTE_LIMIT_API="interruptAttachToTangle, attachToTangle, addNeighbors, removeNeighbors, getNeighbors" \
```

Comment:
In IRI this line would no longer be needed since those are the default values for remote_limit_api since a few versions. Does this count for CLIRI aswell?

Reply:
I don't know, it depends when this was updated in IRI. I recommend keeping this, I don't see the harm in being explicit